src/UIComponents/DataCard.jsx

```diff
@@ -24,6 +25,13 @@ class DataCard extends Component {
       );
     }
 
+    const removedRowsMsg =
+      removedRowsCount > 0
+        ? removedRowsCount === 1
+          ? `${this.props.removedRowsCount} row was removed because it contained an empty cell.`
+          : `${this.props.removedRowsCount} rows were removed because they contained empty cells.`
+        : null;
+
```

Comment:
Possible other way to write this:

```
    const removedRowsMsg =
      removedRowsCount === 1
        ? `${this.props.removedRowsCount} row was removed because it contained an empty cell.`
        : removedRowsCount > 1
        ? `${this.props.removedRowsCount} rows were removed because they contained empty cells.`
        : null;